projects/packages/identity-crisis/src/_inc/admin.css

```diff
@@ -3545,24 +3545,24 @@ https://github.com/thoughtbot/bourbon
   margin-top: 32px;
   text-align: center; }
 
-.jp-recommendations__app-badge {
```

Comment:
Thanks! That makes sense. I'll work to include that in this PR.

Reply:
Hey @jeherve. I took a quick look into it again and realized that CSS - that I don't know whether it's currently maintained, is part of the Jetpack plugin `/build/admin.css` . I'm not sure how much of that file is still used in the IDC package, but from what I could see we can remove pretty much everything related to `jp-recommendations` since I couldn't find any reference to that CSS in the IDC package. 
Based on the comment [here](https://github.com/Automattic/jetpack/blob/master/projects/packages/identity-crisis/src/class-identity-crisis.php#L710), that CSS would need a bit of rework to get it dynamically generated from the style dependencies instead. I'm not sure if somebody is doing or planning to do that in the near future, so should I go ahead and remove only the CSS specific to the `jp-recommendations`?